// @esri/calcite-components/dist/cjs/bs-99e128a2.js
/*!
 * All material copyright ESRI, All Rights Reserved, unless otherwise specified.
 * See https://github.com/Esri/calcite-design-system/blob/main/LICENSE.md for details.
 * v2.1.0
 */
'use strict';

const calciteInputTimePicker_calciteTimePicker_entry = require('./calcite-input-time-picker.calcite-time-picker-d5fe6638.js');
require('./index-02633aec.js');
require('./form-bd8d4323.js');
require('./dom-6bd6bcac.js');
require('./guid-448f3e1e.js');
require('./resources-727c400a.js');
require('./interactive-3760cd3a.js');
require('./browser-e7375f59.js');
require('./key-897cbb95.js');
require('./label-6b071745.js');
require('./component-027255b4.js');
require('./loadable-e9ed2d65.js');
require('./locale-78470e06.js');
require('./observers-fd7323b9.js');
require('./focusTrapComponent-0fe71be6.js');
require('./math-c49a4c80.js');
require('./t9n-92b64378.js');
require('./openCloseComponent-10a646ab.js');
require('./Validation-5cf3ef07.js');

// Bosnian [bs]
var locale = {
  name: 'bs',
  weekdays: 'nedjelja_ponedjeljak_utorak_srijeda_četvrtak_petak_subota'.split('_'),
  months: 'januar_februar_mart_april_maj_juni_juli_august_septembar_oktobar_novembar_decembar'.split('_'),
  weekStart: 1,
  weekdaysShort: 'ned._pon._uto._sri._čet._pet._sub.'.split('_'),
  monthsShort: 'jan._feb._mar._apr._maj._jun._jul._aug._sep._okt._nov._dec.'.split('_'),
  weekdaysMin: 'ne_po_ut_sr_če_pe_su'.split('_'),
  ordinal: function ordinal(n) {
    return n;
  },
  formats: {
    LT: 'H:mm',
    LTS: 'H:mm:ss',
    L: 'DD.MM.YYYY',
    LL: 'D. MMMM YYYY',
    LLL: 'D. MMMM YYYY H:mm',
    LLLL: 'dddd, D. MMMM YYYY H:mm'
  }
};
calciteInputTimePicker_calciteTimePicker_entry.dayjs.locale(locale, null, true);

exports.default = locale;
