// @esri/calcite-components/dist/cjs/mk-5a2f7cae.js
/*!
 * All material copyright ESRI, All Rights Reserved, unless otherwise specified.
 * See https://github.com/Esri/calcite-design-system/blob/main/LICENSE.md for details.
 * v2.1.0
 */
'use strict';

const calciteInputTimePicker_calciteTimePicker_entry = require('./calcite-input-time-picker.calcite-time-picker-d5fe6638.js');
require('./index-02633aec.js');
require('./form-bd8d4323.js');
require('./dom-6bd6bcac.js');
require('./guid-448f3e1e.js');
require('./resources-727c400a.js');
require('./interactive-3760cd3a.js');
require('./browser-e7375f59.js');
require('./key-897cbb95.js');
require('./label-6b071745.js');
require('./component-027255b4.js');
require('./loadable-e9ed2d65.js');
require('./locale-78470e06.js');
require('./observers-fd7323b9.js');
require('./focusTrapComponent-0fe71be6.js');
require('./math-c49a4c80.js');
require('./t9n-92b64378.js');
require('./openCloseComponent-10a646ab.js');
require('./Validation-5cf3ef07.js');

// Macedonian [mk]
var locale = {
  name: 'mk',
  weekdays: 'недела_понеделник_вторник_среда_четврток_петок_сабота'.split('_'),
  months: 'јануари_февруари_март_април_мај_јуни_јули_август_септември_октомври_ноември_декември'.split('_'),
  weekStart: 1,
  weekdaysShort: 'нед_пон_вто_сре_чет_пет_саб'.split('_'),
  monthsShort: 'јан_фев_мар_апр_мај_јун_јул_авг_сеп_окт_ное_дек'.split('_'),
  weekdaysMin: 'нe_пo_вт_ср_че_пе_сa'.split('_'),
  ordinal: function ordinal(n) {
    return n;
  },
  formats: {
    LT: 'H:mm',
    LTS: 'H:mm:ss',
    L: 'D.MM.YYYY',
    LL: 'D MMMM YYYY',
    LLL: 'D MMMM YYYY H:mm',
    LLLL: 'dddd, D MMMM YYYY H:mm'
  },
  relativeTime: {
    future: 'после %s',
    past: 'пред %s',
    s: 'неколку секунди',
    m: 'минута',
    mm: '%d минути',
    h: 'час',
    hh: '%d часа',
    d: 'ден',
    dd: '%d дена',
    M: 'месец',
    MM: '%d месеци',
    y: 'година',
    yy: '%d години'
  }
};
calciteInputTimePicker_calciteTimePicker_entry.dayjs.locale(locale, null, true);

exports.default = locale;
