// @esri/calcite-components/dist/esm/es-89554a4e.js
/*!
 * All material copyright ESRI, All Rights Reserved, unless otherwise specified.
 * See https://github.com/Esri/calcite-design-system/blob/main/LICENSE.md for details.
 * v2.1.0
 */
import { d as dayjs } from './calcite-input-time-picker.calcite-time-picker-e4b7b1aa.js';
import './index-433aebec.js';
import './form-5cf1adb5.js';
import './dom-f74f388a.js';
import './guid-e5232da1.js';
import './resources-46b5a086.js';
import './interactive-8095521e.js';
import './browser-f0a06003.js';
import './key-1abc2b90.js';
import './label-191228ce.js';
import './component-a4952fd5.js';
import './loadable-31b2a807.js';
import './locale-8f40ca35.js';
import './observers-7bffaed0.js';
import './focusTrapComponent-f339292e.js';
import './math-01153984.js';
import './t9n-0c281238.js';
import './openCloseComponent-f1a58cd7.js';
import './Validation-bbd09d59.js';

// Spanish [es]
var locale = {
  name: 'es',
  monthsShort: 'ene_feb_mar_abr_may_jun_jul_ago_sep_oct_nov_dic'.split('_'),
  weekdays: 'domingo_lunes_martes_miércoles_jueves_viernes_sábado'.split('_'),
  weekdaysShort: 'dom._lun._mar._mié._jue._vie._sáb.'.split('_'),
  weekdaysMin: 'do_lu_ma_mi_ju_vi_sá'.split('_'),
  months: 'enero_febrero_marzo_abril_mayo_junio_julio_agosto_septiembre_octubre_noviembre_diciembre'.split('_'),
  weekStart: 1,
  formats: {
    LT: 'H:mm',
    LTS: 'H:mm:ss',
    L: 'DD/MM/YYYY',
    LL: 'D [de] MMMM [de] YYYY',
    LLL: 'D [de] MMMM [de] YYYY H:mm',
    LLLL: 'dddd, D [de] MMMM [de] YYYY H:mm'
  },
  relativeTime: {
    future: 'en %s',
    past: 'hace %s',
    s: 'unos segundos',
    m: 'un minuto',
    mm: '%d minutos',
    h: 'una hora',
    hh: '%d horas',
    d: 'un día',
    dd: '%d días',
    M: 'un mes',
    MM: '%d meses',
    y: 'un año',
    yy: '%d años'
  },
  ordinal: function ordinal(n) {
    return n + "\xBA";
  }
};
dayjs.locale(locale, null, true);

export default locale;
