// @esri/calcite-components/dist/esm/it-91db2f2e.js
/*!
 * All material copyright ESRI, All Rights Reserved, unless otherwise specified.
 * See https://github.com/Esri/calcite-design-system/blob/main/LICENSE.md for details.
 * v2.1.0
 */
import { d as dayjs } from './calcite-input-time-picker.calcite-time-picker-e4b7b1aa.js';
import './index-433aebec.js';
import './form-5cf1adb5.js';
import './dom-f74f388a.js';
import './guid-e5232da1.js';
import './resources-46b5a086.js';
import './interactive-8095521e.js';
import './browser-f0a06003.js';
import './key-1abc2b90.js';
import './label-191228ce.js';
import './component-a4952fd5.js';
import './loadable-31b2a807.js';
import './locale-8f40ca35.js';
import './observers-7bffaed0.js';
import './focusTrapComponent-f339292e.js';
import './math-01153984.js';
import './t9n-0c281238.js';
import './openCloseComponent-f1a58cd7.js';
import './Validation-bbd09d59.js';

// Italian [it]
var locale = {
  name: 'it',
  weekdays: 'domenica_lunedì_martedì_mercoledì_giovedì_venerdì_sabato'.split('_'),
  weekdaysShort: 'dom_lun_mar_mer_gio_ven_sab'.split('_'),
  weekdaysMin: 'do_lu_ma_me_gi_ve_sa'.split('_'),
  months: 'gennaio_febbraio_marzo_aprile_maggio_giugno_luglio_agosto_settembre_ottobre_novembre_dicembre'.split('_'),
  weekStart: 1,
  monthsShort: 'gen_feb_mar_apr_mag_giu_lug_ago_set_ott_nov_dic'.split('_'),
  formats: {
    LT: 'HH:mm',
    LTS: 'HH:mm:ss',
    L: 'DD/MM/YYYY',
    LL: 'D MMMM YYYY',
    LLL: 'D MMMM YYYY HH:mm',
    LLLL: 'dddd D MMMM YYYY HH:mm'
  },
  relativeTime: {
    future: 'tra %s',
    past: '%s fa',
    s: 'qualche secondo',
    m: 'un minuto',
    mm: '%d minuti',
    h: 'un\' ora',
    hh: '%d ore',
    d: 'un giorno',
    dd: '%d giorni',
    M: 'un mese',
    MM: '%d mesi',
    y: 'un anno',
    yy: '%d anni'
  },
  ordinal: function ordinal(n) {
    return n + "\xBA";
  }
};
dayjs.locale(locale, null, true);

export default locale;
