// @esri/calcite-components/dist/esm/pt-br-d033f9fb.js
/*!
 * All material copyright ESRI, All Rights Reserved, unless otherwise specified.
 * See https://github.com/Esri/calcite-design-system/blob/main/LICENSE.md for details.
 * v2.1.0
 */
import { d as dayjs } from './calcite-input-time-picker.calcite-time-picker-e4b7b1aa.js';
import './index-433aebec.js';
import './form-5cf1adb5.js';
import './dom-f74f388a.js';
import './guid-e5232da1.js';
import './resources-46b5a086.js';
import './interactive-8095521e.js';
import './browser-f0a06003.js';
import './key-1abc2b90.js';
import './label-191228ce.js';
import './component-a4952fd5.js';
import './loadable-31b2a807.js';
import './locale-8f40ca35.js';
import './observers-7bffaed0.js';
import './focusTrapComponent-f339292e.js';
import './math-01153984.js';
import './t9n-0c281238.js';
import './openCloseComponent-f1a58cd7.js';
import './Validation-bbd09d59.js';

// Portuguese (Brazil) [pt-br]
var locale = {
  name: 'pt-br',
  weekdays: 'domingo_segunda-feira_terça-feira_quarta-feira_quinta-feira_sexta-feira_sábado'.split('_'),
  weekdaysShort: 'dom_seg_ter_qua_qui_sex_sáb'.split('_'),
  weekdaysMin: 'Do_2ª_3ª_4ª_5ª_6ª_Sá'.split('_'),
  months: 'janeiro_fevereiro_março_abril_maio_junho_julho_agosto_setembro_outubro_novembro_dezembro'.split('_'),
  monthsShort: 'jan_fev_mar_abr_mai_jun_jul_ago_set_out_nov_dez'.split('_'),
  ordinal: function ordinal(n) {
    return n + "\xBA";
  },
  formats: {
    LT: 'HH:mm',
    LTS: 'HH:mm:ss',
    L: 'DD/MM/YYYY',
    LL: 'D [de] MMMM [de] YYYY',
    LLL: 'D [de] MMMM [de] YYYY [às] HH:mm',
    LLLL: 'dddd, D [de] MMMM [de] YYYY [às] HH:mm'
  },
  relativeTime: {
    future: 'em %s',
    past: 'há %s',
    s: 'poucos segundos',
    m: 'um minuto',
    mm: '%d minutos',
    h: 'uma hora',
    hh: '%d horas',
    d: 'um dia',
    dd: '%d dias',
    M: 'um mês',
    MM: '%d meses',
    y: 'um ano',
    yy: '%d anos'
  }
};
dayjs.locale(locale, null, true);

export default locale;
